src/platform/utilities/tests/feature-toggles/helpers.unit.spec.js

```diff
@@ -0,0 +1,68 @@
+import { expect } from 'chai';
+
+const {
+  setCookie,
+  createTokenFromCookie,
+  getFlipperId,
+} = require('../../feature-toggles/helpers');
+
+describe('feature toogles', () => {
+  describe('setCookie', () => {
+    it('should return the cookie if it already exists', () => {
+      const token = 't68mzm0skza1zqwinl4byt';
+      document.cookie = `newcookieName=${token}`;
+      expect(setCookie('newcookieName', 'differentToken')).to.deep.equal(
+        document.cookie.replace(
+          // eslint-disable-next-line no-useless-escape
+          /(?:(?:^|.*;\s*)newcookieName\s*\=\s*([^;]*).*$)|^.*$/,
+          '$1',
+        ),
+      );
+    });
+
+    it('should set a new cookie if one does not already exist', () => {
+      const token = 't68mzm0skza1zqwinl4byt';
+      expect(setCookie('cookieName', token)).to.deep.equal(token);
```

Comment:
This tests whether the function returns the string, not whether a new cookie is added to `document.cookie`.

Also, `.deep.equal()` is for objects. We're comparing strings, so we don't need that `.deep`
```suggestion
      expect(setCookie('cookieName', token)).to.equal(token);
```